### DISCUSSION TITLE
Konsultacja nt. Azure AD Federation + O365 [remote]

### DISCUSSION THREAD

**Discussion by rublon-mw** [AzureCommunityPL/group#195] [Oferty pracy]
Poszukuję eksperta od **Azure AD** oraz **Identity Federation** w kontekście **Office 365** na wstępną konsultację dot. wdrożenia [Rublon MFA](https://rublon.com/doc/o365/). Jest tutaj taka osoba?

Umowa: zlecenie albo usługa (B2B)
Stawka: 100 zł/h
Lokalizacja: Remote

Kontakt: m.wendrowski@adips.com

**Comment by tonyszko** [comment]
A czego dotyczy pytanie - bo ta instrukcja jest raczej step-by-step. 

BTW - z ciekawości czystej, czemu "Rublon" - patrząc na product page, to jego płatna wersja pokrywa się 1:1 z tym co jest wliczone w usługę Azure AD.

**Reply by rublon-mw** [reply]
Hej Tomasz, dziękuję za pytanie.

Przy logowaniu do Office 365 pojawia się następującego typu komunikat po integracji z Rublon:
"AADSTS51004: The user account xxx@xxx.com does not exist in the xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx directory. To sign into this application, the account must be added to the directory."
DirSync u klienta nie działał, ale już działa. Jak identity source jest stosowany lokalny Microsoft Active Directory, który jest synchronizowany z Azure AD.

Klient chce stosować Rublon, aby umożliwić MFA dla użytkowników logujących się do Office 365, Windows oraz VPN (poprzez RADIUS). Nie chce płacić za MFA w Azure.

**Reply by tonyszko** [reply]
A możesz pokazać jak od strony AAD wygląda taki użytkownik, bo to coś nie gra - (.) Jak identity source jest stosowany lokalny Microsoft Active Directory, który jest synchronizowany z Azure AD. (.) 

Jako source powinien być pokazany albo local AAD albo "Synchronized".

Sprawdź jak wyglądają ustawienia federacji/auht w AAD dla xxx.com - ustawienia auth są per-FQDN,  może idzie to gdzieś indziej niż Ci się wydaje. 

Co do decyzji klienta nie będę dyskutował :), ale tak tylko nadmienie:
* MFA w podstawowej formie dla O365/AAD jest za darmo 
* Wsparcie security keys / FIDO jest w standardzie za darmo 
* VPN z Windows przez Azure wspiera wprost uwierzytelnienie przez AAD 
* VPNy innych firm (do sprawdzenia) wspirają wprost uwierzytelnienie w AAD (niekoniecznie trzeba RADIUS) - patrz np. Cisco itp. 

To powyżej jest za darmo, dopiero jak się chce wejść w Conditional Access trzeba zacząć płacić. #justSaying :)
